DorisProject/src/app/app.module.ts

```diff
@@ -40,7 +51,10 @@ import { NumberOnlyDirective } from './pages/directives/number-only.directive';
     ReactiveFormsModule,
     HttpClientModule,
     MaterialModule,
-    BrowserAnimationsModule
+    BrowserAnimationsModule,
+    MatIconModule,
+    MatFormFieldModule,
+    LayoutModule
```

Comment:
Codacy has a fix for the issue: [Insert `,`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9275970)
```suggestion
    LayoutModule,
```